tests/test_templated_prompt.py

```diff
@@ -236,14 +244,19 @@ def test_templated_prompt_builtins(tmp_path_factory):
 
 @pytest.mark.parametrize(
     "questions, raises, returns",
-    (
+    [
         ({"question": {"default": "{{ not_valid }}"}}, None, ""),
         ({"question": {"help": "{{ not_valid }}"}}, None, "None"),
         ({"question": {"type": "{{ not_valid }}"}}, InvalidTypeError, "None"),
         ({"question": {"choices": ["{{ not_valid }}"]}}, None, "None"),
-    ),
+    ],
```

Comment:
There are a few of these minor changes in this PR. I think using a list of parameters is semantically better than using a tuple because in my understanding lists have the semantics of being collections while tuples have the semantics of ordered multi-value data. It's a minor distinction and both work fine technically. Also, [pytest's documentation](https://docs.pytest.org/en/stable/example/parametrize.html) uses a list if that's worth anything. :wink:

Reply:
Let's undo this change too please.

Reply:
Oh well ... Done. :shrug: